modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/images/RoundedCornersTransformation.kt

```diff
@@ -53,7 +53,7 @@ class RoundedCornersTransformation(
         // https://github.com/coil-kt/coil/issues/421#issuecomment-640133205
         val (outputWidth, outputHeight) = input.width to input.height
 
-        val output = createBitmap(outputWidth, outputHeight, input.config)
+        val output = createBitmap(outputWidth, outputHeight, input.config ?: Bitmap.Config.ARGB_8888)
```

Comment:
Changed because of the following error.

> RoundedCornersTransformation.kt:56:62 Argument type mismatch: actual type is 'android.graphics.Bitmap.Config?', but 'android.graphics.Bitmap.Config' was expected.